redis/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM redis:3.2.8
+COPY redis.conf /usr/local/etc/redis/redis.conf
+CMD [ "redis-server", "/usr/local/etc/redis/redis.conf" ]
```

Comment:
Hm, how odd. Perhaps Redis doesn't have a default path then.

It's not a big deal, override the parent image command in that case (as you've done).